.travis.yml

```diff
@@ -32,8 +32,7 @@ env:
   - NPM_TAG=next IMAGE_NAME=theia-swift NODE_VERSION=10
   - NPM_TAG=latest IMAGE_NAME=theia-cpp NODE_VERSION=10.15.3
   - NPM_TAG=next IMAGE_NAME=theia-cpp NODE_VERSION=10.15.3
-  # skip the `rust` CI build due to known failures: https://github.com/theia-ide/theia-apps/issues/253
-  # - NPM_TAG=next IMAGE_NAME=theia-rust NODE_VERSION=10.15.3
+  - NPM_TAG=next IMAGE_NAME=theia-rust NODE_VERSION=10.15.3
```

Comment:
Is there any reason not to have the latest tag as well for rust?

Reply:
Good question. IIRC when this image was introduced, it relied on some "just-released" Theia dependencies, not yet available as `latest`. I think it was `@theia/cpp`. So we could introduce a `latest` version of this image when its CI is "stabilised".